packages/kbn-optimizer/limits.yml

```diff
@@ -61,6 +61,7 @@ pageLoadAssetSize:
   remoteClusters: 51327
   reporting: 183418
   rollup: 97204
+  ruleRegistry: 107971
```

Comment:
Can start this with 100kb?  It looks like it should still pass the limits check.

Reply:
Sure thing 👍